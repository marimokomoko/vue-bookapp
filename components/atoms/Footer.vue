<template>
  <v-footer :absolute="!fixed" app color="purple lighten-3">
    <span>&copy; {{ new Date().getFullYear() }}</span>
  </v-footer>
</template>

<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator'
@Component({})
export default class Footer extends Vue {
  private fixed = false
}
</script>

<style>
</style>