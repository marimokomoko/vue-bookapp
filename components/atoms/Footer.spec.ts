import { createLocalVue, mount, Wrapper } from '@vue/test-utils'
import Vuetify from 'vuetify'
import Footer from '@/components/atoms/Footer.vue'

let wrapper: Wrapper<Vue>;

describe('画面レイアウト', () => {
    it('スナップショットテスト', () => {
        wrapper = mount(Footer, {
            localVue: createLocalVue(),
            vuetify: new Vuetify(),
            stubs: ['router-link']
        })
        expect(wrapper.html()).toMatchSnapshot()
    })
})