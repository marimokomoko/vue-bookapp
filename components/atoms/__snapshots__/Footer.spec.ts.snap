// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`画面レイアウト スナップショットテスト 1`] = `"<footer class=\\"v-footer v-sheet theme--light v-footer--absolute purple lighten-3\\" style=\\"left: 0px; right: 0px; bottom: 0px;\\"><span>© 2022</span></footer>"`;
